dashboard/dashboard/views.py

```diff
@@ -51,17 +52,34 @@ def clean_username(self):
         return username
 
 
+class RacetrackPasswordChangeForm(PasswordChangeForm):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        for visible in self.visible_fields():
+            visible.field.widget.attrs['class'] = 'form-control'
```

Comment:
Hmm, why is this needed?

Reply:
This adds `class` attribute to HTML form elements, giving them a bootstrap style.

Reply:
Ok, can't it be done in static html? ie. I see there `class="form-text"`

Reply:
The problem is that form components are created dynamically in a template (it's part of `change_password.html`)
```
{% for field in form %}
   ...
                {{ field }}
   ...
{% endfor %}
```
so I can't set the class in a static manner. There are more advantages though, the proper components are created automatically by Django based on a default form for changing the password.